controller/user.go

```diff
@@ -0,0 +1,55 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-cluster/app"
+	"github.com/fabric8-services/fabric8-common/auth"
+	"github.com/goadesign/goa"
+
+	"github.com/fabric8-services/fabric8-cluster/application"
+	"github.com/fabric8-services/fabric8-common/log"
+)
+
+// UserController implements the user resource.
+type UserController struct {
+	*goa.Controller
+	app application.Application
+}
+
+// NewUserController creates a user controller.
+func NewUserController(service *goa.Service, app application.Application) *UserController {
+	return &UserController{Controller: service.NewController("UserController"), app: app}
+}
+
+// Clusters runs the clusters action.
+func (c *UserController) Clusters(ctx *app.ClustersUserContext) error {
+	identityID, err := auth.LocateIdentity(ctx)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, err)
+	}
+
+	clusters, err := c.app.IdentityClusters().ListClustersForIdentity(ctx, identityID)
+
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"identity_id": identityID,
+			"err":         err,
+		}, "failed to list clusters for identity %s", identityID)
+		return app.JSONErrorResponse(ctx, err)
+	}
+	data := make([]*app.ClusterData, 0)
+	for _, cluster := range clusters {
```

Comment:
I would suggest we don't use `cluster` as a variable name since it can cause confusion with the existing `github.com/fabric8-services/fabric8-cluster/cluster` pkg.